### ISSUE TITLE
Can't compile any qt5 application (missing include path)

### ISSUE THREAD

**Issue by mcallegari** [Alexpux/MINGW-packages#619]
Hi, I just installed the qt5 package as:
`pacman -S mingw64/mingw-w64-x86_64-qt5`
and I am using the MinGW 64 shell

When I try to build any qt5 based application I got different errors regarding missing includes.

Reproduce with:
`cd /mingw64/share/qt5/examples/quick/`
`qmake`
`make`

The error is like this:
`$ make`
`cd quick-accessibility/ && ( test -e Makefile || C:/msys64/mingw64/bin/qmake.exe C:/msys64/mingw64/share/qt5/examples/quick/quick-accessibility/quick-accessibility.pro -o Makefile ) && make -f Makefile`
`make[1]: ingresso nella directory "/mingw64/share/qt5/examples/quick/quick-accessibility"`
`make -f Makefile.Debug`
`make[2]: ingresso nella directory "/mingw64/share/qt5/examples/quick/quick-accessibility"
g++ -c -pipe -fno-keep-inline-dllexport -march=nocona -mtune=core2 -g -frtti -Wall -Wextra -fexceptions -mthreads -DUNICODE -DQT_QUICK_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_NEEDS_QMAIN -I'.' -I'C:/msys64/mingw64/include/QtQuick' -I'C:/msys64/mingw64/include/QtGui' -I'C:/msys64/mingw64/include/QtQml' -I'C:/msys64/mingw64/include/QtNetwork' -I'C:/msys64/mingw64/include/QtCore' -I'debug' -I'../../../mkspecs/win32-g++'  -o debug/main.o main.cpp`
`In file included from C:/msys64/mingw64/include/QtCore/QDir:1:0,`
`                 from ../shared/shared.h:40,`
`                 from main.cpp:40:`
`C:/msys64/mingw64/include/QtCore/qdir.h:37:28: fatal error: QtCore/qstring.h: No such file or directory`
` #include <QtCore/qstring.h>`
`                            ^`
`compilation terminated.`

Basically the  `-I'C:/msys64/mingw64/include` is missing. 
Weird though because in the Qt5Core.pc file it is there, but it's not taken by Qt.

Any idea ?

**Comment by mcallegari** [comment]
OK, got it.
Using the MSYS2 gcc is NOT OK.
I removed it and installed the mingw64 gcc package and it started to work.

**Comment by mingwandroid** [comment]
Just for clarification purposes.. you didn't need to remove msys2-gcc, you just need to have the mingw-w64 x86_64 one installed and launch your shell via mingw64_shell.bat.

**Comment by mcallegari** [comment]
OK, thanks ! :+1:
